implementations/cats/src/test/scala/sttp/client3/impl/cats/CatsTestBase.scala

```diff
@@ -10,9 +11,7 @@ trait CatsTestBase {
   implicit def executionContext: ExecutionContext
 
   implicit lazy val monad: MonadError[IO] = new CatsMonadAsyncError[IO]
```

Comment:
why do you need this?

Reply:
also unsure. I'll have a deeper look at some point.